en/PlanView/StructureScan.md

```diff
@@ -0,0 +1,37 @@
+# Plan - Structure Scan
+
+A structure scan allows you to create a grid flight pattern which captures images over vertical surfaces. These are tpypically used for the visual inspection or creation of 3d models of structures.
+
+Structure Scan is available to insert into a mission from the Pattern tool.
+
+![](../../images/plan/StructureScan.jpg)
+
+In green you can see the polygon show which represent the structure. In white you can see the flight path for the vehicle. You can also fly circular structure scans by change the polygon to a circle using the center tool.
+
+## Defining the structure shape
+
```

Comment:
Missing text here (unless this is a grouping heading, in which case everything below it needs to be a lower level topic)